crates/autopilot/src/arguments.rs

```diff
@@ -372,6 +380,77 @@ impl std::fmt::Display for Arguments {
     }
 }
 
+/// External solver driver configuration
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct ExternalSolver {
```

Comment:
naming nit (there is no such thing as an internal solver anymore)

```suggestion
pub struct Solver {
```